src/components/NewWorkspaceModal.js

```diff
@@ -106,8 +111,23 @@ const NewWorkspaceModal = Utils.withDisplayName('NewWorkspaceModal', ({
     Ajax(signal).Groups.list().then(setAllGroups)
   ]))
 
+  const loadBucketLocation = _.flow(
+    withErrorReporting('Error loading data'),
+    Utils.withBusyState(setLoading)
+  )(async () => {
+    const [locationResponse] = await Promise.all([
+      Ajax(signal).Workspaces.workspace(namespace, cloneWorkspace.workspace.name).checkBucketLocation(cloneWorkspace.workspace.googleProject, cloneWorkspace.workspace.bucketName)
+    ])
+    setBucketLocation(locationResponse.location)
+  })
+
   // Lifecycle
-  Utils.useOnMount(() => { loadProjectsGroups() })
+  Utils.useOnMount(() => {
+    loadProjectsGroups()
+    if (cloneWorkspace) {
+      loadBucketLocation()
+    }
```

Comment:
We'd typically write this as below for conciseness:
```suggestion
    !!cloneWorkspace && loadBucketLocation()
```

Reply:
Done.